airflow/contrib/hooks/redis_hook.py

```diff
@@ -66,3 +66,47 @@ def get_conn(self):
                 db=self.db)
 
         return self.redis
+
+    def set_key_value(self, k, v):
```

Comment:
```suggestion
    def set_key_value(self, key: str, v: value: Any):
```
Using meaningful names is appreciated. Airflow also encourages adding type annotations ;)

Reply:
hear hear, will do